.github/workflows/cypress-tests-runner.yml

```diff
@@ -60,6 +60,41 @@ jobs:
         if: ${{ env.RUN_TESTS == 'true' }}
         uses: actions/checkout@v4
 
+      - name: Install Node.js
+        if: ${{ env.RUN_TESTS == 'true' }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'npm'
+
+      - name: Install Cypress and dependencies
+        if: ${{ env.RUN_TESTS == 'true' }}
+        run: |
+          npm ci --prefix ./cypress-tests
+
+      - name: Check formatting
+        if: ${{ env.RUN_TESTS == 'true' }}
+        run: |
+          npm run format:check --prefix cypress-tests && npm run lint --prefix cypress-tests
+
+          if [ $? -ne 0 ]; then
```

Comment:
I doubt this would work as expected: I think it would exit the step on first failure.